### PR TITLE
Specify what "object" is when returned from an initializer

### PR DESCRIPTION
**Pull request by koresar**

### CODE REVIEW

**Comment by troutowicz** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -233,7 +233,7 @@ Initializers have the following signature:
 ```
 
 * `options` The `options` argument passed into the stamp, containing properties that may be used by initializers.
-* `instance` The object instance being produced by the stamp. If the initializer returns a different object, it replaces the instance.
+* `instance` The object instance being produced by the stamp. If the initializer returns a different object (as in `_.isObject(obj)`) it replaces the instance.
```
Comment (ericelliott) [inline README.md]: I'm not sure a parenthetical interruption here this is the place to define what is meant by `object`. Maybe after the sentence, instead? Also, I think examples would be better than referencing `_.isObject`, which most people will need to look up.

"(e.g. arrays, functions, objects, regexes, etc...)"

Comment (koresar) [inline README.md]: Yeah. Examples is what the specs lack very much.

**Comment by ericelliott** [comment]
Solved in [PR 53](https://github.com/stampit-org/stamp-specification/pull/53).

**Pull request closed without merging**